lib/smart_answer/calculators/check_benefits_financial_support_calculator.rb

```diff
@@ -163,14 +163,20 @@ def eligible_for_housing_benefit?
     end
 
     def eligible_for_tax_free_childcare?
-      return unless @are_you_working != "no" && @children_living_with_you == "yes"
+      return if @are_you_working == "no"
+      return if @children_living_with_you == "no"
+      return if @children_with_disability == "yes"
```

Comment:
Nit, as this is a method suffixed with a ? these should probably be return false rather than return nil.

Not a blocking comment though.